src/containers/shared/components/Tooltip.tsx

```diff
@@ -0,0 +1,115 @@
+import { CSSProperties } from 'react'
+import { useTranslation } from 'react-i18next'
+import successIcon from '../images/success.png'
+import { localizeDate } from '../utils'
+import '../css/tooltip.scss'
+import PayStringToolTip from '../images/paystring_tooltip.svg'
+import { TxStatus } from './TxStatus'
+import { TxLabel } from './TxLabel'
+import { useLanguage } from '../hooks'
+
+const PADDING_Y = 20
+const DATE_OPTIONS = {
+  hour: 'numeric',
+  minute: 'numeric',
+  second: 'numeric',
+  hour12: true,
+}
+
+export interface TooltipInstance {
+  data?: any
+  mode: string
+  x: number
+  y: number
+}
+
+export const Tooltip = ({ tooltip }: { tooltip?: TooltipInstance }) => {
+  const { t } = useTranslation()
+  const language = useLanguage()
+
+  if (!tooltip) {
+    // eslint-disable-next-line react/jsx-no-useless-fragment
+    return <></>
+  }
+
+  const { data } = tooltip
+
+  const renderNegativeUnlTooltip = () =>
+    data.nUnl.map((key) => {
+      const short = key.substr(0, 8)
+      return <div key={key} className={`nUnl: ${key}`}>{`${short}...`}</div>
+    })
```

Comment:
Should the tooltip be refactored to contain this data as a child element as well?

Reply:
Yeah it could be done further.  Wanted to have this impact the least amount of spots.  This is because I have a future PR that will provide a tooltip hook so that I can properly separate the homepage into components.

Reply:
Making it a sub component is tricky too because it means the tooltip will be rendered ahead of time.  Another solution is to have a property called content which takes a React component.